src/components/learner-credit-management/tests/BudgetDetailPage.test.jsx

```diff
@@ -442,6 +443,40 @@ describe('<BudgetDetailPage />', () => {
     }
   });
 
+  it('renders bne zero state only when the groups feature flag enabled', async () => {
```

Comment:
nit: might be worth negating the test title to indicate more clearly what this is testing; that the bne zero state is NOT rendered with groups feature flag _disabled_, and perhaps also including an assertion for what we _do_ expect to be rendered in the UI for this case.